@xen-orchestra/web/src/types/xo/vbd.type.ts

```diff
@@ -0,0 +1,9 @@
+import type { XoVdi } from '@/types/xo/vdi.type'
+import type { Branded } from '@core/types/utility.type'
+
+export type XoVbd = {
+  id: Branded<'vbd'>
+  name_label: string
+  name_description: string
+  VDI: XoVdi['id']
+}
```

Comment:
Use `XoVbd` from `@vates/types/xo.mts` if this doesn't trigger you too many changes